src/components/navbar/SideBar.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import chessIcon from "../../assets/navbar/Chess.svg";
+import gameControllerIcon from "../../assets/navbar/GameController.svg";
+import houseIcon from "../../assets/navbar/House.svg";
+import infoIcon from "../../assets/navbar/Info.svg";
+import lockKeyIcon from "../../assets/navbar/LockKey.svg";
+import moonStarIcon from "../../assets/navbar/MoonStars.svg";
+import questionIcon from "../../assets/navbar/Question.svg";
+import blankProfile from "../../assets/navbar/blankProfile.svg";
+import Page from "./Page";
+
+interface props {
```

Comment:
please consider changes it to `SideBarProps`
```suggestion
interface SideBarProps {
```